.github/workflows/test.yml

```diff
@@ -1,4 +1,12 @@
-on: push
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
You can remove this line, I just renamed the default branch.

Contributors can run:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
git remote prune origin
```

Reply:
thanks @michaelkirk, done